mult. users active warning

patte opened the pull request:
- put active initials in header
- show a warning if multiple users are active on the same version of a document

<img width="1360" alt="screen shot 2018-03-28 at 17 30 59" src="https://user-images.githubusercontent.com/3500621/38039576-da787842-32ad-11e8-9710-ce2b39954d45.png">
<img width="338" alt="screen shot 2018-03-28 at 17 31 59" src="https://user-images.githubusercontent.com/3500621/38039638-fb2d867c-32ad-11e8-9f94-a3907847d5a3.png">
<img width="1722" alt="screen shot 2018-03-28 at 18 17 53" src="https://user-images.githubusercontent.com/3500621/38042354-734f1c78-32b4-11e8-95c1-598cf062fd81.png">

patte commented:
thank you for the review @tpreusse ! I hoped I addressed the code feedback properly.

To you points.
1. @lukasbuenger It would indeed be very cool if you could look into this
2.
  a) I totally agree, the round trip is less than idea. Do you see a easy way to hook this up to the change detection of the editor @lukasbuenger?
  b) @tpreusse Yes, I know, asking both/all parties on reload if they want to discard changes is not ideal. What would you suggest? Remember it in localstorage? I didn't come up with a smart idea.
3. Is this a option that doesn't require much time @lukasbuenger? If so, it sounds good, if not I prefer to go with this solution. 
4. I thought of that: Timing out edits and cleanup up existing is on the way.

lukasbuenger commented:
1. Will do. I'll just finish my admin stuff first
2. Would only be necessary if we don't do No 3. Generally you want to extend [this](https://github.com/orbiting/publikator-frontend/blob/master/pages/repo/edit.js#L368) to react on changes.
3. It wouldn't take that long. You really only need to pass a Boolean prop to the Slate component and some sensible state machine to unlock the document.

patte commented:
Because this feature is desperately anticipated, I fixed 1 and 2.a. For 2.b/3 I added window.confirm to avoid that users loose their changes.
I think this approach with window.confirm is also better in an educational sense. Let our users understand the situation with a big popup first, and make the experience smoother with readonly later, seems like a better way to rollout this change to me.
Could you @lukasbuenger and @tpreusse please review my changes again.
